Gemfile.lock

```diff
@@ -0,0 +1,179 @@
+GIT
```

Comment:
Also, we are talking about "locking" the dev dependencies here. There are really no runtime deps.